script/validate-data/index.ts

```diff
@@ -54,9 +55,17 @@ async function checkWorkflows(folders: string[]): Promise<WorkflowWithErrors[]>
         const workflowFilePath = join(folder, e.name);
         const propertiesFilePath = join(folder, "properties", `${fileType}.properties.json`)
 
-        const errors = await checkWorkflow(workflowFilePath, propertiesFilePath);
-        if (errors.errors.length > 0) {
-          result.push(errors)
+        const workflowWithErrors = await checkWorkflow(workflowFilePath, propertiesFilePath);
+        if(workflowWithErrors.name) {
+          if(workflow_template_names.includes(workflowWithErrors.name)) {
+            workflowWithErrors.errors.push(`Workflow template name "${workflowWithErrors.name}" already exists`)
+          }
+          else {
+            workflow_template_names.push(workflowWithErrors.name)
+          } 
```

Comment:
I guess Set will simplify this code.

Reply:
> const mySet1 = new Set()
> 
> mySet1.add(1)           // Set [ 1 ]
> mySet1.add(5)        // Set [ 1, 5 ]
> mySet1.add(5)           // Set [ 1, 5 ]

Looking at the above,

`if(mySet1.size == mySet1.add(5).size) {workflowWithErrors.errors.push(`Workflow template name "${workflowWithErrors.name}" already exists`) }`